website/docs/guides/managing-constraints.mdx

```diff
@@ -18,7 +18,7 @@ How large is too large? How complex is too complex? It depends. However, smaller
 
 ## The cost of data
 
-First, let's talk a bit about Unleash's architecture: **Your Unleash instance** is where you define all your features, their strategies, and constraints (and a lot more). However, the Unleash instance itself does not do any of the feature evaluation[^1]. That responsibility is delegated to the Unleash SDKs (or [Edge](../reference/unleash-edge) / the [Unleash proxy](../reference/unleash-proxy)).
+First, let's talk a bit about Unleash's architecture: **Your Unleash instance** is where you define all your features, their strategies, and constraints (and a lot more). However, the Unleash instance itself does not do any of the feature evaluation[^1]. That responsibility is delegated to the Unleash SDKs (or [Edge](/reference/unleash-edge) / the [Unleash proxy](/reference/unleash-proxy)).
```

Comment:
Maybe this is a good time to remove all these links to the old Unleash Proxy?

[This page](https://unleash-docs-git-docs-audit-urls-unleash-team.vercel.app/reference/unleash-proxy) is not really part of the navigation anymore and when you land there you lose the left navigation completely.

Reply:
removed in 85b8c9999c35b89b9567e371a143b5fe67491842